docker/DockerFile.dev

```
@@ -0,0 +1,26 @@
+# check docker/Readme.md
+
+FROM ubuntu:impish
+RUN apt update
+RUN apt upgrade -y
+
+RUN apt-get install -y locales && locale-gen en_US.UTF-8
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+RUN DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt -y install curl build-essential libssl-dev pkg-config lsb-release
```

```suggestion
RUN DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends lsb-release
```
I don't think any of the other deps there are needed.